core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractProject.java

```diff
@@ -133,6 +137,19 @@ public String getAlias() {
   }
 
   @Override
+  public String getShortName() {
+    if (isStringBlankOrNull(shortName)) {
+      return alias;
```

Comment:
This doesn't work since it's for naming and the alias (full project name) is not likely to pass validation when used for naming